en/docs/reference/connectors/connector-usage.md

```diff
@@ -2,37 +2,35 @@
 
 This document provides a set of guidelines on how to use connectors throughout their lifecycle.
 
-## Using connectors in your integration project
+## Use connectors in your integration project
 
-Connectors can be added and used as part of the integration logic of your integration solution. This helps you configure inbound and outbound connections to third-party applications or to systems that support popular B2B protocols.
+Connectors can be added and used as part of the integration logic of your integration solution. This helps you configure inbound and outbound connections to third-party applications or systems that support popular B2B protocols.
 
 ### New connector versions
 
 From time to time there are new connector versions released. These new versions may have new operations and changes to existing operations. When moving to a new connector version from an older version, it is recommended to reconfigure your connector from scratch.
 
-### Importing connectors 
+### Import connectors 
 
-All the connectors are hosted in the [Integration Connector Store](https://store.wso2.com/store/assets/esbconnector/list). You can download the connector from the store as a .zip file. 
+All the connectors are hosted in the [Integration Connector Store](https://store-v2.wso2.com/?page=1&product=MI+Connector). You can download the connector from the store as a .zip file. 
```

Comment:
let's change url to https://store.wso2.com/?page=1&product=MI+Connector

Reply:
Fixed